docs/source/specs/openapi.yml

```diff
@@ -729,6 +771,105 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /api/v1/reports/costs/:
+    get:
+      summary: Query to obtain cost reports
+      operationId: getCostReports
+      tags:
+        - Report
+      parameters:
+        - in: header
+          name: Authorization
+          description: e.g. Token 45138a913da44ab89532bab0352ef84b
+          schema:
+            type: string
+          required: true
+        - in: query
+          name: filter
+          schema:
+            $ref: '#/components/schemas/ReportFilter'
+          description: The filter to apply to the report.
+      responses:
+        '200':
+          description: A report object
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ReportCost'
+        '401':
+          description: Unauthorized
+        '500':
+          description: Unexpected Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /api/v1/reports/inventory/:
+    get:
+      summary: Query to obtain inventory reports
+      operationId: getInventoryReports
+      tags:
+        - Report
+      parameters:
+        - in: header
+          name: Authorization
+          description: e.g. Token 45138a913da44ab89532bab0352ef84b
+          schema:
+            type: string
+          required: true
+        - in: query
+          name: filter
+          schema:
+            $ref: '#/components/schemas/ReportFilter'
+          description: The filter to apply to the report.
+      responses:
+        '200':
+          description: A report object
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ReportInventory'
+        '401':
+          description: Unauthorized
+        '500':
+          description: Unexpected Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /api/v1/reports/storage/:
```

Comment:
I think now that we have the `/inventory/` endpoint, storage will be a portion of the inventory that can be filtered upon.

Reply:
ok. I will remove the storage endpoint and update the resource_scope to add storage types.